### PR TITLE
Polish translation: small fixes 

### PR DESCRIPTION
**Pull request by polishboi**
## Polish translation: small fixes 🇵🇱 ❕ ❗️ ❕ ❗️ ❕ ❗️ ❕ ❗️ ❕ ❗️ ❕ ❗️ 

I've found some translations to be perfect and some to be completely bonkers. 

This is something straight out of Google Translator:
https://github.com/ApolloTVofficial/kamino/blob/522962bcb6f5b868e34137da162e83aa255bfe37/res/values/strings_pl.arb#L33

I'm not trying to be mean.  Some were actually pretty good. Just saying though, someone rushed that translation a bit. Can't find the Polish flag emoji so I'll choose my second most favorite: 🏳️‍🌈

### CODE REVIEW

File: res/values/strings_pl.arb [hunk 1]
```diff
@@ -1,92 +1,92 @@
...
-  "customise_the_theme_and_primary_colors": "nastrajać temat i kolory główne.",
+  "make_appname_yours": "UCZYŃ $appName SWOIM",
+  "appearance": "Wygląd",
+  "customise_the_theme_and_primary_colors": "Dostosuj motyw i kolory podstawowe.",
   "launchpad": "Wyrzutnia",
-  "customise_your_launchpad": "nastrajać wasz Wyrzutnia",
-  "boost_your_experience": "ZWIĘKSZAĆ WASZE DOŚWIADCZENIE",
-  "extensions": "przedłużenie",
+  "customise_your_launchpad": "Dostosuj swoją Wyrzutnię",
+  "boost_your_experience": "WYSTRZEL SWOJE DOZNANIA",
```
Comment (polishboi) [inline res/values/strings_pl.arb]: I've translated it a bit weird. My meaning kind of suggests 'shooting your sensations'. I hope nobody interprets that the wrong way. 💦

Comment (kevincosmos) [reply res/values/strings_pl.arb]: I don't think we really want "shooting your sensations" hahaha - can you suggest something a bit more in-line with "improve/tailor/personalise your experience"

Comment (polishboi) [reply res/values/strings_pl.arb]: I feel proud of myself for this one 😄 "Wystrzel" does not only mean "shoot", but also "launch". I wanted to make a pun about rockets somewhere.

**Comment by NBTX** [comment]
👍

**Pull request merged**